docs/assets/particles-config.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
A few lines what this config is for, also is there a particular format for this json ?

Reply:
This is a JSON file which is a config file for generating animated particles on the Homepage. The format is given in the official docs of [particles.js](https://vincentgarreau.com/particles.js/)

Reply:
just add a link in comments, kind of just tagging all json files so that people coming new to the platform knows why a certain json file exists

Reply:
Sure. Would add the comments.